README.md

```diff
@@ -38,16 +38,15 @@
 
 ## 📣 Idea
 
-[chart.xkcd](https://github.com/timqian/chart.xkcd) is an interesting visualization library written in Javascript, and the chart style of chart.xkcd is so cute that I love it at first sight. 
+[chart.xkcd](https://github.com/timqian/chart.xkcd) is an interesting visualization library written in Javascript, yet the chart style of chart.xkcd is so cute that I love it at first sight. 

```

Comment:
...what the hell about the `yet`. it should not be here.

Reply:
emmm,  will remove it.